dirsrvtests/tests/suites/password/pwdPolicy_attribute_test.py

```diff
@@ -361,6 +363,397 @@ def test_pwdpolicysubentry(topology_st, password_policy):
+        assert False
+
+    if not policy_props:
+        policy_props = {
+            'passwordMinAge': str(days_to_secs(1)),
+            'passwordExp': 'on',
+            'passwordMaxAge': str(days_to_secs(10)),
+            'passwordWarning': str(days_to_secs(3))
+        }
+
+    try:
+        if policy == 'global':
+            pwp_mgr.set_global_policy(policy_props)
+        elif policy == 'subtree':
+            pwp_mgr.create_subtree_policy(dn, policy_props)
+        elif policy == 'user':
+            pwp_mgr.create_user_policy(dn, policy_props)
+        else:
+            raise ValueError(f'Invalid type of password policy: {policy}')
+    except Exception as e:
+        log.fatal(f'Failed to configure {policy} password policy: {e}')
+        assert False
+
+
+def modify_pwp(inst, pwp_mgr, policy, dn=None, policy_props=None):
+    """ Modify password policy """
+    dm = DirectoryManager(inst)
+    dm.rebind()
+
+    if policy != 'global' and not dn:
+        log.fatal('dn is required for nonglobal policy')
+        assert False
+
+    if not policy_props:
+        policy_props = {
+            'passwordMinAge': str(days_to_secs(3)),
+            'passwordMaxAge': str(days_to_secs(30)),
+            'passwordWarning': str(days_to_secs(9))
+        }
+
+    try:
+        if policy == 'global':
+            pwp_mgr.set_global_policy(properties=policy_props)
+        elif policy in ['subtree', 'user']:
+            policy_entry = pwp_mgr.get_pwpolicy_entry(dn)
+            policy_entry.replace_many(*policy_props.items())
+        else:
+            raise ValueError(f'Invalid type of password policy: {policy}')
+        log.info(f'Modified {policy} policy with {policy_props}.')
+    except ldap.LDAPError as e:
+        log.fatal(f'Failed to modify {policy} password policy: LDAPError: {str(e)}')
+        assert False
+    except Exception as e:
+        log.fatal(f'Failed to modify {policy} password policy: Exception: {str(e)}')
+        assert False
+
+@pytest.mark.skipif(ds_is_older('1.3.6'), reason="Not implemented")
+def test_shadowaccount_no_policy(topology_st, shadowUser):
+    """Check shadowAccount under no password policy
+
+    :id: a1b2c3d4-5e6f-7890-abcd-ef1234567890
+    :setup: Standalone instance
+    :steps:
+        1. Add a user with shadowAccount objectclass
+        2. Bind as the user
+        3. Check shadowLastChange attribute is set correctly
+    :expectedresults:
+        1. User is added successfully
+        2. Bind is successful
+        3. shadowLastChange is set correctly (days since epoch)
+    """
+
+    edate = int(time.time() / (60 * 60 * 24))
+
+    log.info(f"Bind as {shadowUser.dn}")
+    shadowUser.bind('password')
+    check_shadow_attr_value(topology_st.standalone, shadowUser.dn,
+                            'shadowLastChange', edate, shadowUser.dn)
+
+
+@pytest.mark.skipif(ds_is_older('1.3.6'), reason="Not implemented")
+def test_shadowaccount_global_policy(topology_st, shadowUser):
```

Comment:
The original test_ticket548_test_global_policy tested shadow attributes on two different users (USER1 and USER2) to verify global policy applies correctly to multiple entries. The ported version only uses one user.